apps/voting/contracts/Voting.sol

```diff
@@ -277,20 +275,23 @@ contract Voting is IForwarder, AragonApp {
     }
 
     function _isVoteOpen(Vote storage vote) internal view returns (bool) {
-        return uint64(now) < (vote.startDate.add(voteTime)) && !vote.executed;
+        return uint64(now) < vote.startDate.add(voteTime) && !vote.executed;
     }
 
     /**
     * @dev Calculates whether `_value` is at least a percentage `_pct` of `_total`
     */
     function _isValuePct(uint256 _value, uint256 _total, uint256 _pct) internal pure returns (bool) {
-        if (_value == 0 && _total > 0)
+        if (_total == 0) {
+            return true;
+        }
+
+        if (_value == 0) {
```

Comment:
I think we should remove this one, but I didn't want to modify original function's behaviour in this commit. What do you think @izqui @sohkai ?

Reply:
I think you can remove this condition; if `_value == 0`, the rest will evaluate to false anyway.

Reply:
I think you can remove this condition; if _value == 0, the rest will evaluate to false anyway.

Reply:
Not if `_pct` is zero too. It's an edge case, but if I wanted to keep the function equivalent I had to add this condition. So we could re-formulate the question as: "If `_pct` is zero, should a `_value` of zero return true or false?" I think it should return true, but the original function would return false.